server/cluster/cluster.go

```diff
@@ -1357,6 +1357,52 @@ func (c *RaftCluster) SlowStoreRecovered(storeID uint64) {
 	c.core.SlowStoreRecovered(storeID)
 }
 
+// NeedAwakenAllRegionsInStore checks whether we should do AwakenRegions operation.
+func (c *RaftCluster) NeedAwakenAllRegionsInStore(storeID uint64) (bool, []uint64) {
+	// TODO: Current checking strategy follows the rule that there
+	// is no more than one slowStore in this cluster. We should upate
+	// it in the future if RaftCluster could tolerate several slowStores.
+	var slowStore *core.StoreInfo
+	slowStoreIDs := make([]uint64, 0)
```

Comment:
Let `slowStoreIds` be nil? in that we don't need to alloc memory for this var when no slow stores

Reply:
I am not sure about how current golang treat a slice with zero capacity